challenge.js

```diff
@@ -8,6 +8,9 @@
  *	Answer below.
  */
 
+ //Open up the inspector using option+CMD+i, add and element using document.createElement() in the command line.
+ //used https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement
```

Comment:
Yes, `document.createElement` will create a new HTML element object. However, it will not attach it to the DOM; it will be a floating, fully-functional HTML element that is only accessible via JavaScript. 1